<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>671</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="player1">
     <attribute name="title">
      <string>Player 1</string>
     </attribute>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>91</width>
        <height>161</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="p1_lab_rob">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Robot</string>
      </property>
     </widget>
     <widget class="QLabel" name="p1_lab_cont">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Controller</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>164</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="p1_lab_rightX">
           <property name="text">
            <string>Right Stick: X-axis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_rightY">
           <property name="text">
            <string>Right Stick: Y-axis</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLCDNumber" name="p1_val_rightX">
           <property name="digitCount">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="p1_val_rightY">
           <property name="digitCount">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>120</y>
        <width>86</width>
        <height>15</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="p1_lab_but">
         <property name="text">
          <string>Button</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p1_val_but">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>156</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="p1_lab_leftX">
           <property name="text">
            <string>Left Stick: X-axis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="p1_lab_leftY">
           <property name="text">
            <string>Left Stick: Y-axis</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLCDNumber" name="p1_val_leftX">
           <property name="digitCount">
            <number>6</number>
           </property>
           <property name="value" stdset="0">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="p1_val_leftY">
           <property name="digitCount">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>156</width>
        <height>108</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="p1_lab_ipAdd">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>IP Address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="p1_val_ipAdd">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="p1_lab_robStat">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Robot Status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="p1_val_robStat">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="p1_lab_robMod">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Robot Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="p1_val_robMod">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="p1_lab_teamCol">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Team Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="p1_val_teamCol">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="p1_conCont">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Link to Controller</string>
      </property>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>line</zorder>
     <zorder>p1_lab_rob</zorder>
     <zorder>p1_lab_cont</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>p1_conCont</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QListWidget" name="debugList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="scanProg">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="clientList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="ipBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="connect2robot">
           <property name="text">
            <string>connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Scan4robot">
           <property name="text">
            <string>Scan for Robots</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>tabWidget</zorder>
   <zorder>scanProg</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
